spec/services/provider_interface/sort_application_choices_spec.rb

```diff
@@ -46,13 +46,21 @@
       described_class.call(application_choices: ApplicationChoice.all).first
     end
 
-    # TODO: groups 1, 3, 9 require the relevant EoC implementation first
+    it '#deferred_offers_pending_reconfirmation' do
+      create(:application_choice, :with_deferred_offer, :previous_year)
+      expect(application_choice.task_view_group).to eq(1)
```

Comment:
Might this be a bit more readable (and maybe reliable) if we assigned the integers to constants?

Reply:
These codes are still very local to `SortApplicationChoices`, so will postpone this for next time we make changes.